Installing unpic in a next.js project causes module not found error

Hey Matt Thanks for all hard work and excited for the future of `unpic` 👍 .

**Describe the bug**
Installing and using `@unpic/react` in a next.js project cause `module not found` error.

**More details:**  I first tried to use `@unpic/react` in my personal project and I got the `Cannot find module`. Then I tried with a next nextjs project  and got `module not found` error. Thought it's not the same module..... I think.

**To Reproduce**
Steps to reproduce the behavior:
1. Install `@unpic/react`
2. use the same example form the next.js example documentation.
3. See sandbox for reference. https://codesandbox.io/p/sandbox/distracted-wood-j0jjcw?file=%2Fpages%2Findex.tsx


**Expected behavior**
To be working without errors

**Screenshots**
![image](https://user-images.githubusercontent.com/32019791/225658105-f74593f4-e715-47a8-94bc-1f4093ad7e15.png)

![image](https://user-images.githubusercontent.com/32019791/225658194-f783a240-1a24-40b6-9950-593e0c8a6bc9.png)

Got the same error. I'm using pnpm and I've got the feeling that this could be related to pnpm module resolution mechanism.

I've setup a fresh repo  (create-next-app + unpic/img) to showcase the issue : https://github.com/clementoriol/unpic-img-pnpm-issue

```
pnp install
pnp dev
```

and you should see the error when loading `localhost:3000` :  
```
Error: Cannot find module '(...)/node_modules/.pnpm/@unpic+react@0.0.20_ld2jel3hspngo3u5lti2kgl2sq/node_modules/next/dist/shared/lib/image-config' imported from (...)/node_modules/.pnpm/@unpic+react@0.0.20_ld2jel3hspngo3u5lti2kgl2sq/node_modules/@unpic/react/dist/next.mjs
Did you mean to import next@13.2.4_biqbaboplfbrettd7655fr4n2y/node_modules/next/dist/shared/lib/image-config.js?
```

--- a/packages/react/src/next.tsx
+++ b/packages/react/src/next.tsx
@@ -1,10 +1,10 @@
 import { useContext, forwardRef, useMemo, useEffect } from "react";
 import type { ImageProps as UnpicImageProps } from "./index";
-import type { ImageConfigComplete } from "next/dist/shared/lib/image-config";
-import { imageConfigDefault } from "next/dist/shared/lib/image-config";
 import { Image as UnpicImage } from "./index";
-import { ImageConfigContext } from "next/dist/shared/lib/image-config-context";
 import { getImageCdnForUrl } from "unpic";
+import type { ImageConfigComplete } from "next/dist/shared/lib/image-config.js";
+import { imageConfigDefault } from "next/dist/shared/lib/image-config.js";
+import { ImageConfigContext } from "next/dist/shared/lib/image-config-context.js";
 
 //
 const configEnv = process.env

